tests/Doc/Bitfield_tutorial.cs

```diff
@@ -0,0 +1,54 @@
+// EXAMPLE: bitfield_tutorial
+// HIDE_START
+
+using NRedisStack.RedisStackCommands;
+using NRedisStack.Tests;
+using StackExchange.Redis;
+
+// HIDE_END
+
+// REMOVE_START
+namespace Doc;
+[Collection("DocsTests")]
+// REMOVE_END
+
+// HIDE_START
+public class Bitfield_tutorial
+{
+
+    [SkipIfRedis(Is.OSSCluster)]
+    public void run()
+    {
+        var muxer = ConnectionMultiplexer.Connect("localhost:6379");
+        var db = muxer.GetDatabase();
+        //REMOVE_START
+        // Clear any keys here before using them in tests.
+        db.KeyDelete("bike:1:stats");
+        //REMOVE_END
+        // HIDE_END
+
+
+        // STEP_START bf
+        long res1 = db.StringIncrement("bike:1:stats", 1000);
```

Comment:
This does not match exactly the Python snippet at https://redis.io/docs/latest/develop/data-types/bitfields/